smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/SymbolVisitor.java

```diff
@@ -124,7 +124,7 @@ public Symbol booleanShape(BooleanShape shape) {
     @Override
     public Symbol listShape(ListShape shape) {
         Symbol reference = toSymbol(shape.getMember());
-        return createSymbolBuilder(shape, format("Array<%s>", reference.getName()), null)
+        return createSymbolBuilder(shape, format("(%s)[]", reference.getName()), null)
```

Comment:
I think a bunch of tests need to be updated for this. Please run `./gradlew test` and fix any issues.

Reply:
Tests updated in https://github.com/awslabs/smithy-typescript/pull/168/commits/64087007ecb05c817a2bf2a99abb983cd0423c2a